lib/components/molecules/Accordian/Accordian.js

```diff
@@ -0,0 +1,62 @@
+// @flow
```

Comment:
Accordian generally works in group; current implementation doesn't handle this. see https://www.w3.org/TR/wai-aria-practices/examples/accordion/accordion.html for an example; you can also explore deatil and summary html5 elements for the same.